### PR TITLE
Fixes #27 - PEP 8 formatting

### LINKED ISSUE #27: PEP8 Enablement including Comments
Make everything PEP8 and increase in-code documentation (comments)

### PR DESCRIPTION
**Pull request by ryescholin**
Fixes #27 

Summary of changes made in this PR:
* PEP 8 formatting
* Removed unused variables

Summary of changes to each file:
* get-image-info.py - Pep 8 formatting
* setup.py - Pep 8 formatting
* objects/hub.py - Pep 8 formatting
* objects/image.py - Pep 8 formatting, removed the following four unused variables: self.num_supported_images, self.sub_images_supported, self.values_exists, self.Charts_exists
* utils/crawler.py - Pep 8 formatting
* utils/setup_utils.py - Pep 8 formatting
* tests/test_utils.py - Pep 8 formatting

Signed-off-by: Rye Scholin

### CODE REVIEW

File: objects/image.py [hunk 1]
```diff
@@ -4,130 +4,129 @@
...
 from utils.crawler import get_repo_pages
 
+
 class App:
-	"""This is the App Name but easier to use mainimage, subimage for documentation
-	These attributes will be from values.yaml (from a tgz)which is derived from
-	the big index.yaml file. Keywords are taken from Chart.yaml which is not saved anywhere.
+	"""This is the App Name but easier to use mainimage, subimage for 
+	   documentation. These attributes will be from values.yaml 
+	   (from a tgz) which is derived from the big index.yaml file.
```
Comment (1ethanhansen) [inline objects/image.py]:
```suggestion
	   (from github) but name is derived from the big index.yaml file. 
```
Updated comments, we no longer use a tarball

File: objects/image.py [hunk 2]
```diff
@@ -4,130 +4,129 @@
...
 
+
 class App:
-	"""This is the App Name but easier to use mainimage, subimage for documentation
-	These attributes will be from values.yaml (from a tgz)which is derived from
-	the big index.yaml file. Keywords are taken from Chart.yaml which is not saved anywhere.
+	"""This is the App Name but easier to use mainimage, subimage for 
+	   documentation. These attributes will be from values.yaml 
+	   (from a tgz) which is derived from the big index.yaml file. 
+	   Keywords are taken from Chart.yaml which is not saved anywhere.
```
Comment (1ethanhansen) [inline objects/image.py]:
```suggestion
	   Keywords are taken from Chart.yaml which is saved in Applications
```

**Review by 1ethanhansen — CHANGES_REQUESTED** [review]
I have just a couple notes, but other than these it looks good!
1. Some of these files still aren't PEP8 compliant. A good resource to quickly check for simple stuff is [this website](http://pep8online.com/). The website isn't perfect (false positives and negatives), but it works pretty well.
The vast majority of the changes that were made look really good and have cleaned up the code a lot.
2. It would be preferable if all of the changes made for this issue are in one commit. As a general rule to make git history easier and cleaner, make one commit for one issue. Use `git rebase -i` to squash commits down to one.
3. On the topic of commits, your commits and PR need a signed-off-by line as specified in [the Contributing Guidlines](https://github.com/mtarsel/ContainerAnalysis/blob/master/docs/CONTRIBUTING.md#legal-stuff).
4. There are some lines where I'm not sure what's going on, so I'll just put the list of permalinks below and if you think there is an actual issue, go ahead and fix it.
* [Should this be tabbed over?](https://github.com/mtarsel/ContainerAnalysis/pull/48/files#diff-621d0abd0405095d2ee41d6a33846661R43)
* [PEP8 says external imports should be together](https://github.com/mtarsel/ContainerAnalysis/pull/48/files#diff-96b13e1c6420f364f41192175f5b38a6R8)
* [Unneeded comment?](https://github.com/mtarsel/ContainerAnalysis/pull/48/files#diff-96b13e1c6420f364f41192175f5b38a6R22)
* [Can we just remove this code?](https://github.com/mtarsel/ContainerAnalysis/pull/48/files#diff-96b13e1c6420f364f41192175f5b38a6R47)
* [tab alignment?](https://github.com/mtarsel/ContainerAnalysis/pull/48/files#diff-2eeaed663bd0d25b7e608891384b7298R7)

It's not quite there yet, but the vast majority of the changes that were made do make the code nicer and brings it closer to PEP8 compliance.

**Comment by mtarsel** [comment]
I would prefer if the changes are in 1 commit

**Review by mtarsel — CHANGES_REQUESTED** [review]
Please make this a single commit.

File: tests/test_utils.py [hunk 3]
```diff
@@ -1,68 +1,76 @@
-import pytest
+mport pytest
```
Comment (1ethanhansen) [inline tests/test_utils.py]:
```suggestion
import pytest
```

**Review by 1ethanhansen — CHANGES_REQUESTED** [review]
Looks much better! There are still a few things, but we can just clean those up as we work on those lines of code. So I have just two quarrels:
1. With the new code here ibm-ace-dashboard-dev and ibm-powervc-k8s-volume-driver both come up with "Image is bad" in results, when they should come up with 3 and 2 images respectively.
2. In tests/test_utils.py there was a reason for the empty lines. Two lines signifies a break between tests that come from different files and just one line if for normal breaks between test functions. I'd like to keep that distinction unless you have a good reason to get rid of it.
Once those issue are fixed and it is a single commit, should be good to go!

**Comment by 1ethanhansen** [comment]
@ryescholin do you know why travis isn't building for this pull request? It looks like travis isn't configured for your fork of the repo, which could be why some of these issues are slipping through

**Review by ryescholin — COMMENTED** [review]
makes sense, i dont know about travis

**Comment by mtarsel** [comment]
I merged a pr in this morning so I dont believe this pr branch is current with master which is why travis was failing. I would prefer this is in a single commit but nonetheless, this branch should be fetched from upstream master.

**Review by 1ethanhansen — CHANGES_REQUESTED** [review]
Needs to be one commit
use `git rebase -i HEAD~11`

**Comment by 1ethanhansen** [comment]
Still not good to merge yet - this should be 13 bad apps, this PR gives 15
https://travis-ci.com/mtarsel/ContainerAnalysis/builds/121445569#L256

**Comment by mtarsel** [comment]
This PR has grown and I would prefer it is only a single commit

**Pull request merged**